scalecube-configuration/src/main/java/io/scalecube/configuration/authorization/DefaultPermissions.java

```diff
@@ -0,0 +1,26 @@
+package io.scalecube.configuration.authorization;
+
+import io.scalecube.account.api.Role;
+import io.scalecube.configuration.api.ConfigurationService;
+import io.scalecube.security.api.Authorizer;
+
+public class DefaultPermissions {
+
+  public static final Authorizer PERMISSIONS =
+      Permissions.builder()
+          .grant(
+              ConfigurationService.CONFIG_CREATE_REPO, Role.Owner.toString(), Role.Admin.toString())
```

Comment:
Mayne let's use enum for roles instead of strings?

Reply:
we could use just map for that and to set priorities to the reoles, so that if member has access to the url, it is automatically allowed to admin and owner.

like, Owner(2) -> Admin(1) - Member(0)

Reply:
this can be nice improvment